### PR TITLE
ITEA-15-make-an-html-page-draft

### PR DESCRIPTION
**Pull request by RomanPryima**
The name of branch was changed. I did nothing with files from the previous commit.

### CODE REVIEW

File: homework-2/set-tuple-dict.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (UkrainianEngineer) [inline homework-2/set-tuple-dict.py]: Is this a part of your page?

File: travel/Html/base.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+{% load staticfiles %}
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
```
Comment (UkrainianEngineer) [inline travel/Html/base.html]: Please use just 2 spaces in your HTML and CSS code.

File: travel/Html/base.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="description" content="Boocking system">
+		<meta name="keywords" content="Travel,travels,boocking,tickets,hotels">
+		<title>Travel platform</title>
+	<link href="https://fonts.googleapis.com/css?family=Acme|Wendy+One" rel="stylesheet">	
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="{% static 'css/style.css' %}">
+	
+<body class="container smallscr">
+	{% block content %}
```
Comment (UkrainianEngineer) [inline travel/Html/base.html]: Please fix formatting for `content` block. Something is wrong with spaces around.

File: travel/config_reader.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+"""
```
Comment (UkrainianEngineer) [inline travel/config_reader.py]: Looks like config parsed is another task. It should be merged into separate branch.

**Review by UkrainianEngineer — CHANGES_REQUESTED** [review]

File: travel/Html/main.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+		</a>
+		</div>
+		<div class="col-md-4 choose" id="transport"><a href="https://lviv.itea.ua/">
+		<img class="img-responsive" src="pics/bus.jpg" alt="bus">
+		<span class="text-center">Transport</span>
+		</a>
+		</div>
+		<div class="col-md-4 choose" id="restaurants"><a href="https://lviv.itea.ua/">
+		<img class="img-responsive" src="pics/restoran.jpg" alt="restoran">
+		<span class="text-center">Retaurants</span>
```
Comment (Sasha1152) [inline travel/Html/main.html]: missing letter "s"

File: travel/Html/css/style.css [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+.container{
+	background-color:rgba(0,139,139,.4);	
+	background: -webkit-linear-gradient(top,rgba(0,139,139,1),rgba(0,0,0,0)); /*Safari 5.1-6*/
+	background: -o-linear-gradient(bottom,rgba(0,139,139,1),rgba(0,0,0,0)); /*Opera 11.1-12*/
+	background: -moz-linear-gradient(bottom,rgba(0,139,139,1),rgba(0,0,0,0)); /*Fx 3.6-15*/
+	background: linear-gradient(to bottom, rgba(0,139,139,1), rgba(0,0,0,0)); /*Standard*/
+}
+.head-bkg{
+	background-color: white;
```
Comment (pivancy) [inline travel/Html/css/style.css]: wtf?

File: travel/config_reader.py (hunk 4 above)
Comment (RomanPryima) [reply travel/config_reader.py]: Removed in local repository. Made push origin. But files are still in remote branch. Will check on the next lesson.

File: travel/Html/main.html (hunk 5 above)
Comment (RomanPryima) [reply travel/Html/main.html]: Fixed

File: travel/Html/base.html (hunk 2 above)
Comment (RomanPryima) [reply travel/Html/base.html]: Removed extra spaces.

File: homework-2/set-tuple-dict.py (hunk 1 above)
Comment (RomanPryima) [reply homework-2/set-tuple-dict.py]: Removed

File: travel/Html/base.html (hunk 3 above)
Comment (RomanPryima) [reply travel/Html/base.html]: fixed

File: travel/config_reader.py (hunk 4 above)
Comment (RomanPryima) [reply travel/config_reader.py]: Already done

File: travel/Html/base.html [hunk 7]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+{% load staticfiles %}
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta name="description" content="Boocking system">
```
Comment (UkrainianEngineer) [inline travel/Html/base.html:8]: typo

File: travel/Html/base.html [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+<meta name="keywords" content="Travel,travels,boocking,tickets,hotels">
```
Comment (UkrainianEngineer) [inline travel/Html/base.html:9]: typo

File: travel/Html/css/style.css [hunk 9]
```diff
@@ -0,0 +1,72 @@
+
+body{
+	font-family: 'Acme', sans-serif;
```
Comment (UkrainianEngineer) [inline travel/Html/css/style.css:3]: too many spaces

File: travel/Html/css/style.css [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+
+.white {
+	color: white;
+}
+.row {
+	margin-top: 15px
+}
+.choose {
+	padding-top: 10px;
+	display:inline-block;
```
Comment (UkrainianEngineer) [inline travel/Html/css/style.css:43]: Something happened with spaces

File: travel/Html/css/style.css [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+	margin-top: 15px
+}
+.choose {
+	padding-top: 10px;
+	display:inline-block;   
+    position:relative;
+}
+
+.choose span {
+    display:inline-block;
```
Comment (UkrainianEngineer) [inline travel/Html/css/style.css:48]: space after `:` in a lot of places

File: travel/Html/main.html [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+	</a>
+	</div>
+	<div class="col-md-4 choose" id="attractions"><a href="https://lviv.itea.ua/">
+	<img class="img-responsive" src="pics/attraction.jpg" alt="attraction">
+	<span class="text-center">Attractions</span>
+	</a>
+	</div>
+	<div class="col-md-4 choose" id="entertaiment"><a href="https://lviv.itea.ua/">
+	<img class="img-responsive" src="pics/entertainment.jpg" alt="entertaiment">
+	<span class="text-center">Entertaiment</span>
```
Comment (UkrainianEngineer) [inline travel/Html/main.html:29]: typo

File: travel/Html/main.html [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+	<span class="text-center">Retaurants</span>
+	</a>
+	</div>
+</div>
+</div>
+
+<div class="row">
+	<div class="col-md-6 text-center"><h2>How to get there?</h2></div>
+	<div class="col-md-4">		
+	<form action="<!---->" method="get">
```
Comment (UkrainianEngineer) [inline travel/Html/main.html:53]: what is the action here?

File: travel/Html/main.html [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+	<h3><input class="form-control text-center" id="location" type="text" value="enter your location"></h3>
+	</div>
+	<div class="col-md-2">
+	<button type="submit" class="btn btn-default"><h3>Submit</h3></button>
+	</form>
+	</div>
+</div>
+
+<div class="row">
+	<div class="text-right col-md-2"><h3>I have</h3></div>
```
Comment (UkrainianEngineer) [inline travel/Html/main.html:63]: too many spaces

File: travel/Html/travel_platform.html [hunk 15]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="description" content="Boocking system">
```
Comment (UkrainianEngineer) [inline travel/Html/travel_platform.html:7]: same typo

File: travel/Html/travel_platform.html [hunk 16]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```
Comment (UkrainianEngineer) [inline travel/Html/travel_platform.html:1]: I can't catch a difference between this file and `main.html`

File: travel/Html/main.html (hunk 5 above)
Comment (Sasha1152) [reply travel/Html/main.html]: :+1:

File: travel/Html/main.html (hunk 13 above)
Comment (RomanPryima) [reply travel/Html/main.html:53]: @pivancy  action will be defined later. Now it is awry commented

File: travel/Html/travel_platform.html (hunk 16 above)
Comment (RomanPryima) [reply travel/Html/travel_platform.html:1]: The main.html is a block for inserting into base.html in Django.
The travel_platform is full html, as it was developed.

File: travel/Html/main.html [hunk 17]
```diff
@@ -0,0 +1,94 @@
...
+	<div class="col-md-2">
+	<form action="<!---->" method="get">
```
Comment (kate-ka) [inline travel/Html/main.html:65]: I can't see no closing form tag

**Comment by Sasha1152** [comment]
:+1:

File: travel/Html/main.html (hunk 13 above)
Comment (UkrainianEngineer) [reply travel/Html/main.html:53]: Then it's much better to remove action at all.

File: travel/Html/travel_platform.html (hunk 16 above)
Comment (UkrainianEngineer) [reply travel/Html/travel_platform.html:1]: Well, if you're trying to implement logic with base template, you don't need to write full HTML in separate file. You just need to extend base template with some specific information.

**Review by UkrainianEngineer — CHANGES_REQUESTED** [review]

**Comment by UkrainianEngineer** [comment]
@RomanPryima why this pull request was closed?

**Comment by RomanPryima** [comment]
created new branch. As we decided

**Pull request closed without merging**